### PR TITLE
finish static pages

### PR DESCRIPTION
**Pull request by d2luu**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,47 +1,36 @@
...
-
-# Use ActiveModel has_secure_password
-# gem 'bcrypt', '~> 3.1.7'
-
-# Use Unicorn as the app server
-# gem 'unicorn'
-
-# Use Capistrano for deployment
-# gem 'capistrano-rails', group: :development
+gem 'rails', 									 '4.2.6'
```
Comment (toxingau) [inline Gemfile]: double quote

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,12 @@
...
+  # Returns the full title on a per-page basis.
+  def full_title(page_title = '')
+    base_title = t "static_pages.title"
+    if page_title.empty?
+      base_title
+    else
+      page_title + " | " + base_title
+    end
+  end
+end
```
Comment (dat-hedspi) [inline app/helpers/application_helper.rb]: end file

File: app/views/layouts/_footer.html.erb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<footer class="footer">
+  <small>
+    The <%= link_to  t("static_pages.title"), root_path %>
+  </small>
+  <nav>
+    <ul>
+      <li><a href="http://news.railstutorial.org/" target = '_blank'>News</a></li>
```
Comment (dat-hedspi) [inline app/views/layouts/_footer.html.erb]: I18n

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    <%= link_to t("static_pages.title"), root_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to t("home"),   home_path %></li>
+        <li><%= link_to t("login"), '#' %></li>
+      </ul>
+    </nav>
+  </div>
+</header>
+
```
Comment (dat-hedspi) [inline app/views/layouts/_header.html.erb]: eof

Comment (toxingau) [inline app/views/layouts/_header.html.erb]: EOF

File: Gemfile (hunk 1 above)
Comment (qmau94) [inline Gemfile]: double quote

File: app/helpers/application_helper.rb [hunk 5]
```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
-end
+
+  # Returns the full title on a per-page basis.
+  def full_title(page_title = '')
```
Comment (qmau94) [inline app/helpers/application_helper.rb]: double quote

File: app/views/layouts/_footer.html.erb (hunk 3 above)
Comment (qmau94) [inline app/views/layouts/_footer.html.erb]: i18n

File: app/views/layouts/_header.html.erb (hunk 4 above)
Comment (qmau94) [inline app/views/layouts/_header.html.erb]: eof

File: app/views/layouts/_header.html.erb [hunk 6]
```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to t("static_pages.title"), root_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to t("home"),   home_path %></li>
+        <li><%= link_to t("login"), '#' %></li>
```
Comment (qmau94) [inline app/views/layouts/_header.html.erb]: double quote

File: app/views/layouts/_header.html.erb [hunk 7]
```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to t("static_pages.title"), root_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to t("home"),   home_path %></li>
```
Comment (qmau94) [inline app/views/layouts/_header.html.erb]: ident

File: app/views/layouts/_shim.html.erb [hunk 8]
```diff
@@ -0,0 +1,4 @@
+<!--[if lt IE 9]>
+  <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/r29/html5.min.js">
+  </script>
+<![endif]-->
```
Comment (qmau94) [inline app/views/layouts/_shim.html.erb]: eof

File: app/views/layouts/application.html.erb [hunk 9]
```diff
@@ -1,14 +1,18 @@
...
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
-  <%= csrf_meta_tags %>
-</head>
-<body>
-
-<%= yield %>
-
-</body>
+  <head>
+    <title><%= full_title(yield(:title)) %></title>
```
Comment (qmau94) [inline app/views/layouts/application.html.erb]: bỏ ngoặc

File: app/views/layouts/application.html.erb [hunk 10]
```diff
@@ -1,14 +1,18 @@
...
+    <%= csrf_meta_tags %>
+    <%= stylesheet_link_tag    'application', media: 'all',
```
Comment (qmau94) [inline app/views/layouts/application.html.erb]: double quote

File: app/views/layouts/application.html.erb [hunk 11]
```diff
@@ -1,14 +1,18 @@
...
+                                              'data-turbolinks-track': 'reload' %>
```
Comment (qmau94) [inline app/views/layouts/application.html.erb]: ident

File: app/views/static_pages/home.html.erb [hunk 12]
```diff
@@ -0,0 +1,11 @@
+<% provide :title, t("home") %>
+<div class="center jumbotron">
+  <h1><%= t"welcome" %></h1>
+
+  <h2>
+    <%= t "static_pages.content_1" %> 
+		<%= link_to t("static_pages.link_to"), "http://www.railstutorial.org/", target: '_blank' %>
```
Comment (qmau94) [inline app/views/static_pages/home.html.erb]: ident

File: config/routes.rb [hunk 13]
```diff
@@ -1,56 +1,5 @@
...
-  #   resources :photos, concerns: :toggleable
-
-  # Example resource route within a namespace:
-  #   namespace :admin do
-  #     # Directs /admin/products/* to Admin::ProductsController
-  #     # (app/controllers/admin/products_controller.rb)
-  #     resources :products
-  #   end
+  root 'static_pages#home'
+  get '/home', to: 'static_pages#home'
```
Comment (qmau94) [inline config/routes.rb]: get "home" => "static_pages#home"

File: app/helpers/application_helper.rb [hunk 14]
```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
+
+  # Returns the full title on a per-page basis.
+  def full_title(page_title = "")
+    base_title = t "static_pages.title"
+    if page_title.empty?
```
Comment (qmau94) [inline app/helpers/application_helper.rb]: page_title.empty? ? base_title : page_title + " | " + base_title

File: app/views/layouts/_footer.html.erb [hunk 15]
```diff
@@ -0,0 +1,10 @@
+<footer class="footer">
+  <small>
+    The <%= link_to  t("static_pages.title"), root_path %>
```
Comment (qmau94) [inline app/views/layouts/_footer.html.erb]: ident

File: app/views/layouts/_footer.html.erb [hunk 16]
```diff
@@ -0,0 +1,10 @@
...
+  </small>
+  <nav>
+    <ul>
+      <li><a href="http://news.railstutorial.org/" target = '_blank'><%= t "news" %></a></li>
```
Comment (qmau94) [inline app/views/layouts/_footer.html.erb]: qua 80 chars

File: app/views/layouts/application.html.erb [hunk 17]
```diff
@@ -1,14 +1,18 @@
...
-</head>
-<body>
-
-<%= yield %>
-
-</body>
+  <head>
+    <title><%= full_title(yield :title) %></title>
+    <%= csrf_meta_tags %>
+    <%= stylesheet_link_tag    "application", media: "all",
```
Comment (qmau94) [inline app/views/layouts/application.html.erb]: ident

File: app/views/static_pages/home.html.erb [hunk 18]
```diff
@@ -0,0 +1,10 @@
+<% provide :title, t("home") %>
+<div class="center jumbotron">
+  <h1><%= t"welcome" %></h1>
+  <h2>
+    <%= t "static_pages.content_1" %> 
+		<%= link_to t("static_pages.link_to"),
```
Comment (qmau94) [inline app/views/static_pages/home.html.erb]: ident

File: Gemfile [hunk 19]
```diff
@@ -1,47 +1,36 @@
 source 'https://rubygems.org'
```
Comment (hellovietnam93) [inline Gemfile]: nháy kép

File: Gemfile [hunk 20]
```diff
@@ -1,47 +1,36 @@
...
+gem "coffee-rails", "4.1.0"
+gem "jquery-rails", "4.0.3"
+gem "turbolinks", "2.3.0"
+gem "jbuilder", "2.2.3"
+gem "sdoc", "0.4.0", group: :doc
 
 group :development, :test do
-  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug'
+	gem "sqlite3", "1.3.9"
```
Comment (hellovietnam93) [inline Gemfile]: ident, 1 tab = 2 cách thôi em

File: app/helpers/application_helper.rb [hunk 21]
```diff
@@ -1,2 +1,8 @@
 module ApplicationHelper
+
+  # Returns the full title on a per-page basis.
```
Comment (hellovietnam93) [inline app/helpers/application_helper.rb]: xóa remark đi em nhé

File: app/helpers/application_helper.rb [hunk 22]
```diff
@@ -1,2 +1,8 @@
...
+  def full_title(page_title = "")
```
Comment (hellovietnam93) [inline app/helpers/application_helper.rb]: bỏ ()

File: app/views/layouts/application.html.erb [hunk 23]
```diff
@@ -1,14 +1,18 @@
...
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
-  <%= csrf_meta_tags %>
-</head>
-<body>
-
-<%= yield %>
-
-</body>
+  <head>
+    <title><%= full_title(yield :title) %></title>
```
Comment (hellovietnam93) [inline app/views/layouts/application.html.erb]: bỏ ()

File: app/views/layouts/application.html.erb [hunk 24]
```diff
@@ -1,14 +1,18 @@
...
+    <%= csrf_meta_tags %>
+    <%= stylesheet_link_tag "application", media: "all",
+    	"data-turbolinks-track": "reload" %>
+    <%= javascript_include_tag "application", "data-turbolinks-track": "reload" %>
+    <%= render "layouts/shim" %>
+  </head>
+  <body>
+  	<%= render "layouts/header" %>
```
Comment (hellovietnam93) [inline app/views/layouts/application.html.erb]: ident

File: app/views/static_pages/home.html.erb [hunk 25]
```diff
@@ -0,0 +1,10 @@
+<% provide :title, t("home") %>
+<div class="center jumbotron">
+  <h1><%= t"welcome" %></h1>
+  <h2>
+    <%= t "static_pages.content_1" %> 
+    <%= link_to t("static_pages.title"), root_path, target: "_blank" %>
+    <%= t "static_pages.content_2" %>
+  </h2>
+  <%= link_to "Log in", '#', class: "btn btn-lg btn-primary" %>
```
Comment (hellovietnam93) [inline app/views/static_pages/home.html.erb]: I18n cho log in + ""

File: config/routes.rb [hunk 26]
```diff
@@ -1,56 +1,5 @@
...
-  #   resources :posts, concerns: :toggleable
-  #   resources :photos, concerns: :toggleable
-
-  # Example resource route within a namespace:
-  #   namespace :admin do
-  #     # Directs /admin/products/* to Admin::ProductsController
-  #     # (app/controllers/admin/products_controller.rb)
-  #     resources :products
-  #   end
+  root 'static_pages#home'
```
Comment (hellovietnam93) [inline config/routes.rb]: nháy kép nhé

**Pull request merged**